UnitTests/CommonTestUtils/ConfigureJoinableTaskFactoryAttribute.cs

```diff
@@ -21,12 +21,15 @@ public sealed class ConfigureJoinableTaskFactoryAttribute : Attribute, ITestActi
 
         public ActionTargets Targets => ActionTargets.Test;
 
+        public ConfigureJoinableTaskFactoryAttribute()
+        {
+            Application.ThreadException += HandleApplicationThreadException;
```

Comment:
That's probably better.
We did find a bug in WinForms implementation that under heavy load led to race conditions and overall instability
https://github.com/dotnet/winforms/pull/2713